### PR TITLE
Certificate pinning for download server

### PR DESCRIPTION
**Pull request by haosap**
This PR resolved the issue mentioned in EXPOSUREAPP-1838. 
In this PR, a new preprocessor `ENABLE_WHITELIST`  is introduced. We don't remove the white list directly, because we need it to do some debug while developing

### CODE REVIEW

**Comment by ChristianKienle** [comment]
I would like to test this using a mim-scenario. Give please don't merge before that is done. This will be the next thing on my plate. <3

**Review by ChristianKienle — APPROVED** [review]
Approved - builds are failing though. @haosap can you check as to why?

**Comment by haosap** [comment]
Hi @ChristopherSchmitz ,
Please review it. Thanks.

**Review by ChristopherSchmitz — APPROVED** [review]

**Comment by ChristopherSchmitz** [comment]
@haosap understood that we keep the whitelist. Since we have the compilation flag active the impact to the security side (shipment of non production code) is mitigated.

Did we tested the functionality locally against int, wru and prod?

Chris

**Review by ChristopherSchmitz — COMMENTED** [review]
Need confirmation that this was tested locally against int, WRU and prod with whitelist flag deactive

**Comment by haosap** [comment]
> I would like to test this using a mim-scenario. Give please don't merge before that is done. This will be the next thing on my plate. <3

Hi @ChristianKienle ,
I think you mentioned in the call that you will test it locally?

File: src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.strings [hunk 1]
```diff
@@ -987,5 +987,5 @@ Daha fazla bilgi için lütfen SSS sayfamıza bakın.";
  during the last 14 days then `Active_Tracing_Interval_Partially_Active` is used – otherwise
  `Active_Tracing_Interval_Partially_Active` will be used.
  */
-"Active_Tracing_Interval_Partially_Active" = "Maruz kalma günlüğü son %1$u günde %2$u etkindi.";
+"Active_Tracing_Interval_Partially_Active" = "Maruz kalma günlüğü son %2$u günde %1$u gün etkindi.";
```
Comment (inf2381) [inline src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.strings:990]: Could you please revert this unintended change?

**Review by ChristianKienle — APPROVED** [review]

**Comment by haosap** [comment]
Hi @ChristopherSchmitz ,
@ChristianKienle tested the landscapes locally already. I think you can approve it again.
Thanks.

**Review by ChristopherSchmitz — APPROVED** [review]

**Comment by ChristopherSchmitz** [comment]
> Hi @ChristopherSchmitz ,
> @ChristianKienle tested the landscapes locally already. I think you can approve it again.
> Thanks.

approved

**Pull request merged**